packages/paste-website/src/pages/tokens/design-tokens-package.mdx

```diff
@@ -91,8 +101,10 @@ As you use Paste, you'll likely encounter things that _don't seem right_. Please
 
 ***
 
-<Box marginTop="space90">
+<Box marginTop="space90" id="changelog">
```

Comment:
Why does this need an id?

Reply:
Left over `id` from when I was adding the changelog anchor to the TOCs. Just forgot to remove.